simtools/tests/test_visualize.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+import logging
+import os
+import sys
+import numpy as np
+from astropy import units as u
+from astropy.io import ascii
+from .. import visualize
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+
+def test_plot_1D():
+
+    logger.debug('Testing plot1D')
+
+    xTitle = 'Wavelength [nm]'
+    yTitle = 'Mirror reflectivity [%]'
+    headersType = {'names': (xTitle, yTitle),
+                   'formats': ('f8', 'f8')}
+    title = 'Test 1D plot'
+
+    dataIn = np.loadtxt('tests/testData/ref_200_1100_190211a.dat',
+                        usecols=(0, 1), dtype=headersType)
+
+    # Change y-axis to percent
+    if '%' in yTitle:
+        if np.max(dataIn[yTitle]) <= 1:
+            dataIn[yTitle] = 100*dataIn[yTitle]
+    data = dict()
+    data['Reflectivity'] = dataIn
+    for i in range(5):
+        newData = np.copy(dataIn)
+        newData[yTitle] = newData[yTitle]*(1 - 0.1*(i + 1))
+        data['{}%% reflectivity'.format(100*(1 - 0.1*(i + 1)))] = newData
+
+    plt = visualize.plot1D(data, title=title, palette='autumn')
+
+    plotFile = 'tests/testPlots/plot_1D.pdf'
+    if os.path.isfile(plotFile):
```

Comment:
Can we try to use only pathlib for paths for consistency?
You can do:
`from pathlib import Path`
`if Path(plotFile).exists():`

Reply:
Sure, but is there a specific reason to use pathlib?

Reply:
Not a strong one. I've been using it since the beginning and it never gave me a reason to change.